app/models/cti/caller_id.rb

```diff
@@ -2,6 +2,8 @@ module Cti
   class CallerId < ApplicationModel
     self.table_name = 'cti_caller_ids'
 
+    DefaultCountryId = '49'
```

Comment:
I think we should define this via the settings / depending on the location of of zammad instance / admin like the translations. What do you think?